app/components/Settings/Preview/TableTree.react.js

```diff
@@ -27,6 +22,31 @@ class TableTree extends Component {
         })
     }
 
+    /**
+     *
+     * @param {object} props - Component Props
+     * @param {object} props.schemaRequest - Schema Request
+     * @param {function} props.getSqlSchema - Get SQL Schema Function
+     * @param {function} props.updatePreview - Update Preview Function
+     * @param {object} props.preview - Preview details
+     * @param {object} props.connectionObject - Connection Object
+     * @param {object} props.connectionObject.database - Database Name
+     * @param {object} props.connectionObject.dialect - Dialect for DB
+     * @param {object} props.connectionObject.storage - Storage
+     */
+    constructor(props) {
+        super(props);
+        this.storeSchemaTree = this.storeSchemaTree.bind(this);
+    }
+
+    /**
+     * Gets the label for tree
+     * @param {object} connectionObject - Connection Object
+     * @param {object} connectionObject.database - Database if not dialect not Data World or SQLite
+     * @param {object} connectionObject.storage - If dialect is SQLLite
+     * @param {object} connectionObject.url - If dialect is Dataworld
+     * @returns {string} label
+     */
     getLabel(connectionObject) {
```

Comment:
Since you're working on this file. Please, could refactor this function so that it doesn't take any arguments? i.e. let's use `switch (this.props.connectionObject.dialect)`.